core/controllers/learner_dashboard_test.py

```diff
@@ -333,3 +366,57 @@ def test_anonymous_feedback_is_recorded_correctly(self):
 
         self.assertEqual(messages_summary['author_username'], None)
         self.assertEqual(messages_summary['author_picture_data_url'], None)
+
+    def test_get_with_suggestions_update_suggestion_summary(self):
```

Comment:
This test name is pretty confusing, perhaps `test_get_suggestions_after_updating_suggestion_summary` instead?

Reply:
Done